v03_pipeline/lib/methods/sex_check.py

```diff
@@ -23,30 +23,6 @@
 )
 
 
-class Ploidy(Enum):
-    AMBIGUOUS = 'ambiguous_sex'
-    FEMALE = 'XX'
-    MALE = 'XY'
-
-
-def annotate_discrepant_sex(
```

Comment:
This logic (which was ported directly from the methods scripts) is happening within [this function now](https://github.com/broadinstitute/seqr-loading-pipelines/pull/578/files#diff-81bda30e9edc96dbaf6b9533d4db17f8afa17df35de4b37c3c05e1856e6779dbR157)